src/database/search_handler.h

```diff
@@ -429,7 +429,7 @@ class SearchParser {
     std::shared_ptr<ASTNode> parseRelationshipExpression();
     std::shared_ptr<ASTNode> parseParenthesis();
     std::shared_ptr<ASTQuotedString> parseQuotedString();
-    void checkIsExpected(TokenType tokenType, const std::string& tokenTypeDescription);
+    void checkIsExpected(TokenType tokenType, std::string_view tokenTypeDescription);
 
```

Comment:
That method is not called anywhere. Drop it.

Reply:
Removed.